.github/workflows/coding-standards.yml

```diff
@@ -5,14 +5,16 @@ on:
     branches:
       - "v*.*"
       - "master"
+      - "feature/*"
```

Comment:
Do you really want to run the CI on PR *targetting* a feature branch ?

Reply:
Sure, if such a PR were to exist.